Felienne/hedy
# Hide the is-teacher checkbox if the user uses an invite code #1382

**Description**

Remove the is-teacher checkbox when the user is using a teacher invite code

**Fix for**

#1382 

**How to test**

Before testing locally, you need to run `build-tools/heroku/generate-typescript`
- Go to the sign up page and verify that the checkbox is present.
- Add the environment variable TEACHER_INVITE_CODE:test and go to /invite/test url. Then choose to create a new account and check that the is-teacher checkbox is not present.
- When the change is deployed to Alpha, check that when a user is using the an invite code, a single email will be sent for teacher's approval.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
